package.json

```diff
@@ -72,7 +72,7 @@
     "@types/bootstrap": "^4.4.1",
     "@types/color-convert": "^2.0.0",
     "@types/deep-equal": "^1.0.1",
-    "@types/eslint": "^7.22.0",
+    "@types/eslint": "^8.0.0",
```

Comment:
Any reason not to specify the latest version 8.4.10?

Reply:
Guess not